go.mod

```diff
@@ -8,9 +8,9 @@ require (
 	github.com/docker/docker v20.10.25+incompatible
 	github.com/docker/go-units v0.5.0
 	github.com/gin-gonic/gin v1.9.1
-	github.com/go-vela/sdk-go v0.20.2-0.20230925160153-e6031fe704d7
-	github.com/go-vela/server v0.20.1-0.20230925142408-ab85e96f27c4
-	github.com/go-vela/types v0.20.2-0.20230922185343-b83bcddfa60d
+	github.com/go-vela/sdk-go v0.20.2-0.20230926162005-fe9cd4fb6060
+	github.com/go-vela/server v0.20.1-0.20230926163613-a22924dc271b
+	github.com/go-vela/types v0.20.2-0.20230926151838-42959d67d753
```

Comment:
this seems off.. we should be at v0.20.3+

Reply:
v0.20.3+ doesn't seem to have the latest change of v21 work so errors were being thrown out

Reply:
yea, i feel like the notation should be `github.com/go-vela/sdk-go v0.20.3-0.20230926162005-fe9cd4fb6060` pointing at the latest available tag followed by time and sha. however, the docs say:
> Note: As a best practice, always allow Go tools to generate the pseudo-version number rather than creating your own.